app/src/main/java/com/picketing/www/databases/inmemory/InmemoryDataStore.java

```diff
@@ -0,0 +1,51 @@
+package com.picketing.www.databases.inmemory;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Component
+@RequiredArgsConstructor
+public class InmemoryDataStore implements DataStore {
+
+    private final Map<String, Map<Long, Object>> store;
+    private final Map<String, Long> sequenceStore;
+
+    @Override
+    public Long save(String tableName, Object value) {
+        Map<Long, Object> table = store.get(tableName);
+        if (table == null) {
+            store.put(tableName, new ConcurrentHashMap<>());
+            table = store.get(tableName);
+        }
```

Comment:
`computeIfAbsent`로 더 간단히 할 수 있습니다.
```suggestion
        Map<Long, Object> table = store.computeIfAbsent(tableName, k -> new ConcurrentHashMap<>());
```